site/en/models.md

```diff
@@ -0,0 +1,105 @@
+# Models for OCR-D processors
+
+OCR engines rely on pre-trained models for their recognition. Every engine has
+its own internal format(s) for models. Some support central storage of models
+at a specific location (tesseract, ocropy, kraken) while others require the full
+path to a model (calamari).
+
+Likewise, model distribution is not currently centralised within OCR-D though we
+are working towards a central model repository.
+
+In the meantime, this guide will show you, for each OCR engine:
+
+  * which types of models are supported
+  * where to store models locally
+  * which currently available models we recommend
+  * how to invoke the resp. OCR-D wrapper for the engine with a specific model
+
+## Tesseract / ocrd_tesserocr
+
+Tesseract models are single files with a `.traineddata` extension.
+
+Tesseract expects models to be in a directory `tessdata` within what Tesseract
+calls `TESSDATA_PREFIX`. When installing Tesseract from Ubuntu packages, that
+location is `/usr/share/tesseract-ocr/4.00/tessdata`. When building from source
+using [ocrd_all](htttps://github.com/OCR-D/ocrd_all), the models are searched
+at `/path/to/ocrd_all/venv/share/tessdata`. If you want to override the
+locations, you can set the `TESSDATA_PREFIX` environment variable, e.g. if you
+want the models location to be `$HOME/tessdata`, you can by adding to your
+`$HOME/.bashrc`: `export TESSDATA_PREFIX=$HOME`.
+
+We recommend you download the following models, either by downloading and
+saving to the right location or by running `make install-models-tesseract` when
+using `ocrd_all`:
+
+  * [equ](https://github.com/tesseract-ocr/tessdata_fast/raw/master/equ.traineddata)
+  * [osd](https://github.com/tesseract-ocr/tessdata_fast/raw/master/osd.traineddata)
+  * [eng](https://github.com/tesseract-ocr/tessdata_fast/raw/master/eng.traineddata)
+  * [deu](https://github.com/tesseract-ocr/tessdata_fast/raw/master/deu.traineddata)
+  * [frk](https://github.com/tesseract-ocr/tessdata_fast/raw/master/frk.traineddata)
+  * [script/Latin](https://github.com/tesseract-ocr/tessdata_fast/raw/master/script/Fraktur.traineddata)
+  * [script/Fraktur](https://github.com/tesseract-ocr/tessdata_fast/raw/master/script/Fraktur.traineddata)
```

Comment:
I don't know whether this detail should be covered in the guide, but when installing models via dpkg, there's no `script/` prefix...

Reply:
Good point, I'll add a note on that.